release_notes/ocp_3_6_release_notes.adoc

```diff
@@ -0,0 +1,2471 @@
+different types of services from different providers, while service providers
+benefit from having one integration point that gives them access to multiple
+platforms.
+
+[IMPORTANT]
+====
+This feature is not for production workloads and is currently not fully
+supported. This is using an alpha version of the Kubernetes service catalog.
+====
+
+This feature consists of:
+
+- The Service Consumer: The individual, application , or service that uses a service enabled by the broker and catalog.
+- The Catalog: Where services are published for consumption.
+- Service Broker: Publishes services and intermediates service creation and credential configuration with a provider.
+- Service Provider: The technology delivering the service.
+- Open Service Broker API: Lists services, provisions and deprovisions, binds, and unbinds.
+
+See
+xref:../install_config/install/advanced_install.adoc#enabling-service-catalog[Enabling
+the Service Catalog] for more information.
+
+[[ocp-36-ansible-service-broker]]
+==== Ansible Service Broker (Technology Preview)
+
+- Implementation of the open service broker API that enables users to leverage Ansible
+for provisioning and managing of services via the service catalog on {product-title}.
+- Standardized approach for delivering simple to complex multi-container
+{product-title} services.
+- Works in conjunction with Ansible playbook bundles (APB), which is a lightweight
+meta container comprised of a few named playbooks for each open service broker
+API operations.
+
+Service catalog and Ansible service broker must be configured during
+{product-title} installation. Once enabled, APB services can be deployed right
+from Service Catalog UI.
+
+See
+xref:../install_config/install/advanced_install.adoc#configuring-ansible-service-broker[Configuring
+the Ansible Service Broker] for more information.
+
+[WARNING]
+====
+The Ansible service broker is currently a Technology Preview feature and
+should not be used in production clusters.
+====
+
+[[ocp-36-ansible-playbook-bundles]]
+==== Ansible Playbook Bundles (APB) (Technology Preview)
+
+Ansible playbook bundles (APB) (currently in
+xref:ocp-36-technology-preview[Technology Preview]) is a short-lived, lightweight
+container image consisting of:
+
+* Simple directory structure with named action playbooks
+* Metadata consisting of:
+** required/optional parameters
+** dependencies (provision versus bind)
+* Ansible runtime environment
+* Leverages existing investment in Ansible playbooks and roles
+* Developer tooling available for guided approach
+* Easily modified or extended
+* Example APB services included with {product-title} 3.6:
+** MediaWiki, PostgreSQL
+
+When a user orders an application from the service catalog, the Ansible service
+broker will download the associated APB image from the registry and run it. Once
+the named operation has been performed on the service, the APB image will then
+terminate.
```

Comment:
IMO, here, the 2 sub sections `Ansible Service Broker (Technology Preview)` and `Ansible Playbook Bundles (APB) (Technology Preview)` are more related with and under former section `Installation` than section `Web Console`.
Because, in fact, they belong to ansible installation and the link here `See Configuring the Ansible Service Broker for more information` link to url `... install_config/install/advanced_install.html`

So better to put these 2 sub sections under section `Installation`. @jianlinliu @mdshuai better to double confirm this.

But here `Web Console` section can do a simple mention of `Example APB services MediaWiki, PostgreSQL` somewhere, e.g. under sub section `Search Catalog (Technology Preview)`, whose functions can search the APB services, to make Web Console and APB connected.